proposals/migrate-cli.md

```diff
@@ -0,0 +1,68 @@
+# Migrate Bigchaindb cli for Tendermint
+
+## Problem Description
+With Tendermint integration some of the cli sub-commands have been rendered obsolete. It would be only appropriate to white list sub-commands depending on the backend.
+
+### Use cases
+- Avoid confusing the user by not displaying irrelevant sub-commands.
+
+
+## Proposed Change
```

Comment:
Some other sub-commands might require modification. For example, `bigchaindb configure` shouldn't allow `rethinkdb` or `mongodb` if this is BigchainDB-with-Tendermint.

Reply:
In case we have both 1.x and 2.0 in the master then we would have to allow the user to configure the database with either of options i.e. `rethinkdb`, `mongodb`, `localmongodb`. Once configured the sub-commands will be available depending on the configured backend.

Reply:
:+1: